test/unit/Cover/buyCover.js

```diff
@@ -11,9 +11,9 @@ const gracePeriodInDays = 120;
 const NXM_ASSET_ID = 255;
 
 const buyCoverFixture = {
+  coverId: 0,
```

Comment:
is this still relevant?

Since we're pulling the id from the event and the param to buyCover is MaxUint256

Reply:
updated